debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java

```diff
@@ -133,6 +133,7 @@ public void shouldConsumeRecordsFromInsert() throws Exception {
         assertThat(routedEvent.topic()).isEqualTo("outbox.event.User");
 
         Struct valueStruct = requireStruct(routedEvent.value(), "test payload");
+        assertThat(valueStruct.schema().name()).isEqualTo("test_server.outboxsmtit.outbox.Value-outbox");
```

Comment:
The reporter of the issue suggested this:

> The expected outcome would be that debezium register schemas in the registry with the proper name and namespace taken from the database.server.name and aggregate type or something similar right ?

I would agree with this, i.e. I don't see much value really in that configurable suffix, but adding the aggregate type seems reasonable. In particular, it sets us up nicely for later on, when the actual payload structure (and its schema) is more apparent on the Kafka record level (see DBZ-1297). I.e. here we might have a schema name like `test_server.outboxsmtit.outbox.User.Value`. We might also consider excluding the outbox table name `test_server.outboxsmtit.User.Value`, but then the name might collide with other tables and/or event types from other outbox tables in the same schema. So probably not a good idea.

Reply:
Could you also add an assertion for the _key_ schema?